### PR TITLE
Update value, gasPrice, gasLimit types

### PR DESCRIPTION
**Pull request by listenaddress**

### CODE REVIEW

**Comment by listenaddress** [comment]
I believe this is close to ready, but we need Zondax to update their serialization. Getting "Unexpected data type" error. 
Screen Shot 2020-03-30 at 6 44 19 PM

File: components/Wallet/Send.js/index.jsx [hunk 1]
```diff
@@ -125,10 +125,10 @@ const Send = ({ close }) => {
       const message = new Message({
         to: toAddress,
         from: wallet.address,
-        value: new BigNumber(value.fil.toAttoFil()).toFixed(0, 1),
-        method: 0,
-        gasPrice: gasPrice.toAttoFil(),
-        gasLimit: gasLimit.toAttoFil(),
+        value: new BigNumber(value.fil.toAttoFil()),
+        method: 1,
```
Comment (Schwartz10) [inline components/Wallet/Send.js/index.jsx]: whoa, is 1 the new transfer?

File: store/states.test.js [hunk 2]
```diff
@@ -685,14 +685,14 @@ describe('state manipulators', () => {
   describe('populateRedux', () => {
     test('it sets pending messages in state', () => {
       const pendingMsgs = []
-      for (let i = 0; i < 10; i += 1) {
+      for (let i = 1; i < 11; i += 1) {
```
Comment (Schwartz10) [inline store/states.test.js]: lol this doesn't really matter, but why?

**Comment by Schwartz10** [comment]
@listenaddress this looks good, thank you for putting it together. Do we need to wait until testnet/3 to merge this? (and obviously any changes on juan's end)

**Comment by listenaddress** [comment]
Ya we need to wait for Juan's changes.

**Pull request closed without merging**